resource-managers/cook/src/main/scala/org/apache/spark/scheduler/cluster/cook/CoarseCookSchedulerBackend.scala

```diff
@@ -335,6 +336,12 @@ class CoarseCookSchedulerBackend(
       .disableMeaCulpaRetries()
       .setRetries(1)
 
+    // If we're allocating a static cluster, make a single group to help cook do group
+    // scheduling optimization and get them launched together.
+    if (!schedulerContext.isDynamicAllocationEnabled) {
```

Comment:
You will get scheduled up to 10 cpus. The group scheduling will try to order the queue so that all your executors are scheduled near the same time, it won't prevent executors from being scheduled scheduling the whole group will exceed your quota.

Reply:
If your quota is 10 cpus and you request a job (Spark executor) with 20 cpus, the job will be rejected. Did you mean quota of 100 cps and spark requests 6 executors with 20 cpus each?